tooling/lib/flatten-schema.js

```diff
@@ -44,10 +48,17 @@ module.exports = async (schema, oas) => {
         if (value.type === 'object') {
           array.push(flattenSchema(value, getName(parent, prop), level + 1));
         } else if (value.type === 'array' && value.items) {
```

Comment:
Will evaluating value.type === 'array' always result in value.items being an array? Another way to interpret whether value.items is an array would be `Array.isArray(value.items)`

Reply:
`value.items` is always going to be an object as any other type attached to `items` is not valid against the OAS.

Reply:
Ok!